common/preference/src/main/java/com/my/kizzy/preference/Prefs.kt

```diff
@@ -89,10 +89,11 @@ object Prefs {
         val lastDeleted = get(
             key = LAST_DELETED,
             // Force delete everyone's saved images for the first time
-            defaultValue = System.currentTimeMillis() - 24.hours.inWholeMilliseconds
+            // Same time as Cloudflare cache invalidation
+            defaultValue = System.currentTimeMillis() - 7.days.inWholeMilliseconds
         )
         val currentTime = System.currentTimeMillis()
-        if (currentTime - lastDeleted > 24.hours.inWholeMilliseconds) {
+        if (currentTime - lastDeleted > 7.days.inWholeMilliseconds) {
```

Comment:
This is okay but some people might still be using the older way so dont change this until very far in future